### PR TITLE
Try to fix cronjob PR creation

### PR DESCRIPTION
**Pull request by oli-obk**
The main tests use `bash` several times, maybe that is the issue here, too?

### CODE REVIEW

**Comment by RalfJung** [comment]
Lol, we perfectly raced. ;) but https://github.com/rust-lang/miri/pull/2946 was a bit earlier.
@bors r-

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -195,8 +195,10 @@ jobs:
           git config --global user.name 'The Miri Conjob Bot'
           git config --global user.email 'miri@cron.bot'
       - name: get changes from rustc
+        shell: bash
         run: ./miri rustc-pull
```
Comment (RalfJung) [inline .github/workflows/ci.yml:199]: I think the issue is that the Miri repo is not checked out in this job.

Also, don't you have to install rustup-toolchain-install-master somewhere?

Comment (oli-obk) [reply .github/workflows/ci.yml:199]: force pushed a change that should do everything needed

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -196,6 +197,11 @@ jobs:
           git config --global user.email 'miri@cron.bot'
       - name: get changes from rustc
         run: ./miri rustc-pull
+      - name: Install rustup-toolchain-install-master
+        if: ${{ steps.cache.outputs.cache-hit != 'true' }}
+        shell: bash
+        run: |
+          cargo install -f rustup-toolchain-install-master
```
Comment (RalfJung) [inline .github/workflows/ci.yml:204]:
```suggestion
        run: cargo install -f rustup-toolchain-install-master
```
we don't have a cache here so the rest doesn't do much (and no idea why we set `shell`...)

**Pull request merged**